src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp

```diff
@@ -1512,7 +1512,10 @@ namespace OpenMS
       if (ret == QMessageBox::Cancel) return; // Escape was pressed
       if (ret == QMessageBox::Yes)
       {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Can you please add a comment on which warning this is suppressing and why or when it can be removed.